rules/framework/vfs_overlay.bzl

```diff
@@ -14,7 +14,7 @@ VFSOverlayInfo = provider(
 # Make roots for a given framework. For now this is done in starlark for speed
 # and incrementality. For imported frameworks, there is additional search paths
 # enabled
-def _make_root(ctx, framework_name, root_dir, extra_search_paths, module_map, hdrs, private_hdrs, has_swift):
+def _make_root(ctx, framework_name, root_dir, extra_search_paths, module_map, swiftmodule, hdrs, private_hdrs, has_swift):
```

Comment:
TODO this swiftmodule is misleading, need to rename everywhere

Reply:
Do we need to expose swiftinfo for the imports? I'm wondering if that's already happening elsewhere and we just need to pull it out